benchmark/sparse/linear_regression/training_benchmark.md

```diff
@@ -0,0 +1,87 @@
+# Linear Regression Benchmark Results 
+
+## Summary
+```
+   Results below show the performance comparison of linear regression with MXNet vs Keras-Tensorflow using sparse tensors
+```                                                   
+
+### Results
+### Training Benchmark
+### Configuration
+| Dataset          | Synthetic(Randomly generated)                                |
+| :--------------- | :----------------------------------------------------------- |
+| Keras            | v2.2.2                                                      |
+| TensorFlow       | v1.11.0                                                     |
+| MXNet-mkl         | v1.3.0   
+
+###### Using 10 epochs
+#### CPU
+##### Speed
+###### Note speed calculated using an average of 5 runs
+| Instance Type | GPUs  | Batch Size  | Keras-MXNet (Time/Epoch) | Keras-TensorFlow (Time/Epoch)  |
+|-----|-----|-----|-----|-----|
+| C5.8XLarge |   0  | 64  | 715.4 sec | 676.6 sec
+| C5.8XLarge |   0  | 128 | 346.2 sec | 341.1 sec 
+| C5.8XLarge |   0  | 256 | 168.9 sec | 165.5
+| C5.8XLarge |   0  | 512 | 89.3 sec | 83.8 sec 
+| C5.8XLarge |   0  | 1024 | 50.5 sec | 48.6 sec
+
+#### Memory consumed
```

Comment:
We should have memory in MB as well (like you have for inference)

Reply:
Added